src/common/store/background-epics.ts

```diff
@@ -0,0 +1,31 @@
+import { filter, mergeMap } from 'rxjs/operators';
+import { combineEpics, Epic, ofType } from 'redux-observable';
+import { locateAnnotations } from 'common/store/tabs/tab/annotations/actions';
+import { retrieveActionTab, syncTabMark } from 'common/store/tabs/action-tab';
+import { tabLocateAnnotations } from 'background/messages';
+import { syncBadgeWithAnnotations } from 'background/badge';
+import * as endpoints from '../api/endpoints';
+
+interface FluxStandardAction {
+  type: string | symbol | any;
+  payload?: any;
+  error?: boolean | any;
+  meta?: any;
+}
+
+export const locateEpic: Epic<FluxStandardAction> = action$ => action$.pipe(
+  ofType('API_READ'),
+  filter(action => action.payload.endpoint === endpoints.ANNOTATIONS),
+  mergeMap(async (action) => {
+    const tabId = retrieveActionTab(action);
+    const locationData = await tabLocateAnnotations(tabId, action.payload.data)
+    console.log(locationData);
```

Comment:
Either get rid of this console log or if it was intentional, make this log more descriptive.